### ISSUE TITLE
Looping parameter for animation

### ISSUE THREAD

**Issue by MichalSZZ** [photopea/photopea#1225]
That is required functionality when you want to make banners for Google Ads (or other advertising networks).
In their specification, animation must stop after 30 secounds.
In Photoshop there is a possibility (in timeline/frame editor or in export panel) to set number of loops (x times, once, forever).
When I make the banners I must calculate time for all frames per loop. 
For example:

- Frame 1: Text 1 (2s)
- Frame 2: Text 2 (2s)
- Frame 3: Text 3 (0.5s) + CTA button
- Frame 4: Text 3 (0.5s) + CTA button little darker (blinking, but without stroboscop effect ;))
- Frame 5: Text 3 (0.5s) + CTA button
- Frame 6: Text 3 (0.5s) + CTA button little darker (blinking, but without stroboscop effect ;))

In summary thats together give to me 6s and can be played 5 times in loop to get 30s at all.
After that it must be stopped in last frame.

I think that it will be great to have that functionality in "save for web" window.

![ekampanie-970x250-za-darmo-30s](https://user-images.githubusercontent.com/38118273/52595213-9f363480-2e4d-11e9-8bda-c159b226bbd3.gif)

**Comment by photopea** [comment]
Hey, I added a new parameter Repeat to the Save For Web, that is shown only when exporting an animation.

It works for exporting GIF, PNG (APNG) and WEBP.

**Comment by MichalSZZ** [comment]
WOW. Awesome job!
Do you want to be better than Adobe? 
Just put information about how long GIF will be played :)

In export for the web window, next to place where user can type how many times GIF should be looped. I see, that there is a space for that.

![gif-export](https://user-images.githubusercontent.com/38118273/54745691-6d418c00-4bca-11e9-98d3-ae363a8f1807.png)


e.g. from my last comment
(2s + 2s + 0.5s + 0.5s + 0.5s + 0.5s) and user will set 5x in input. On the right (only when !=0) it should calculate the time. In that case 30s (sometimes it will be number with comma)

It is hard to implement that feature?

**Comment by photopea** [comment]
I think 90% of people would want Forever repeating GIFs. Shouldn't we show the duration of one loop instead?

**Comment by MichalSZZ** [comment]
Hehe. 90% of ppl or more.
That was my first (fast) thing, how to do that. 
In photoshop I hate when must to calculate how long will be played one loop (on timeline there is that information, but in frame editor there isn't)

**Comment by MTyson** [comment]
Damn, Photopea is one of THE awesome tools out there
